topics/proteomics/tutorials/metaquantome-data-creation/tutorial.md

```diff
@@ -0,0 +1,646 @@
+>
+>
+{: .hands_on}
+
+## *Extracting EC values*
+
+> ### {% icon hands_on %} Hands-on: Extract EC numbers
+The cut tool cuts out specific columns from the dataset. In this case, the cut tool is being used to extract columns 1 (peptide) and 3 (EC number) from the dataset peptinfo EC.tsv output. This is a manipulation tool for metaQuantome's convinience.
+>
+> 1. {% tool [Cut](Cut1) %} with the following parameters:
+>    - *"Cut columns"*: `c1,c3`
+>    - {% icon param-file %} *"From"*: `output_ec_tsv` (output of **Unipept** {% icon tool %})
+> 2. Rename EC_table.
+{: .hands_on}
+
+
+## *Filtering confident peptides*
+
+Query Tabular is a tool that can load tabular data into a SQLite database. This step precedes UniPept, as a list containing the peptide sequences must be generated. In this step a list of gene ontology (GO) terms is being generated.
+
+> ### {% icon hands_on %} Hands-on: Filtering confident peptides
+>
+> 1. {% tool [Query Tabular](toolshed.g2.bx.psu.edu/repos/iuc/query_tabular/query_tabular/3.0.0) %} with the following parameters:
+>    - In *"Database Table"*:
+>        - {% icon param-repeat %} *"Insert Database Table"*
+>            - {% icon param-file %} *"Tabular Dataset for Table"*: `output_go_tsv` (output of **Unipept** {% icon tool %})
+>            - In *"Table Options"*:
+>                - *"Specify Name for Table"*: `Goterm`
+>                - *"Specify Column Names (comma-separated list)"*: `peptide,total_protein_count,go_term,protein_count,go_name,go_funct`
+>    - *"SQL Query to generate tabular output"*:
+```
+SELECT Goterm.*
+FROM Goterm
+WHERE ((1.0*Goterm.protein_count)/(1.0*Goterm.total_protein_count)) >= 0.05
+```
+>    - *"include query result column headers"*: `Yes`
+>
+> 2. Rename as Unipept_Function.
+>
+>    > ### {% icon comment %} Comment
+>    >
+>    > In the Unipept API output, the threshold is set to 0.5% of the overall number of peptides unambiguously assigned to a taxon at a particular taxonomic rank level. Here in the Galaxy platform, we are using Query tabular to perform this filtering.
+>    {: .comment}
+>
+{: .hands_on}
+
+
+## *Removing Hashtag from output*
+
+This step is to remove the hashtag from the Peptide header in the Unipept output.
+
+> ### {% icon hands_on %} Hands-on: Remove # from peptide header
+> 1. {% tool [Replace Text](toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_replace_in_line/1.1.2) %} with the following parameters:
+>    - {% icon param-file %} *"File to process"*: `Unipept_Function` (output of **Unipept** {% icon tool %})
+>    - In *"Replacement"*:
+>        - {% icon param-repeat %} *"Insert Replacement"*
+>            - *"Find pattern"*: `#peptide`
+>            - *"Replace with:"*: `peptide`
+>
+> 2. Rename file as All_functions
+{: .hands_on}
+
+
+## *Filter* - EC values
+
+We are using this Query tabular ot rename the output that we obtained from the Cut column tool.
```

Comment:
ot -> to?

And I guess this tool does more than rename output from the cut tool? maybe have a look at this?